modern-business/style-editor.scss

```diff
@@ -10,6 +10,28 @@ Modern Business Editor Styles
 @import "sass/navigation/menu-main-navigation";
 @import "sass/typography/headings";
 
+/** === Helper Functions === */
+
+/**
+ Given a string, $alignment, returns the nested FSE block selectors
+ required for Full Site Editing to look correct. If $alignment is given,
+ we use it to target wp-blocks with a matching data-align attribute.
+ If no alignment is given, we target the base wp-block class. In both
+ of these scenarios, we target top level blocks, and any top level blocks
+ inside one of the FSE template part blocks.
+ */
+@function get-block-selector($alignment) {
```

Comment:
Note that we usually include these functions in a separate file called `_functions.scss`.

Also, you can use [annotations](http://sassdoc.com/annotations/) to change the structure of the docstring and specify the params/returns:

```css
/**
 * Get the nested FSE block selectors
 * required for Full Site Editing to look correct. If $alignment is given,
 * we use it to target wp-blocks with a matching data-align attribute.
 * If no alignment is given, we target the base wp-block class. In both
 * of these scenarios, we target top level blocks, and any top level blocks
 * inside one of the FSE template part blocks.
 * @param {String} $alignment - Alignment
 * @return {String}
 */
```

Reply:
Good call! I'll make sure to add those if we stick with this function